themes/spichka/resources/styles/front-page.scss

```diff
@@ -0,0 +1,74 @@
+.section-banner-video {
+  height: 100vh;
+  align-items: center;
+  display: flex;
+
+  .banner-video {
+    position: absolute;
+    inset: 0;
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    filter: brightness(60%);
+  }
+
+  .header {
+    color: white;
+    position: relative;
+    z-index: 2;
+
+    p {
```

Comment:
a good practice is to use classes and not tag names. But if it is more difficult in wordpress then ok

Reply:
Yes, sure. Will refactor this.